src/DynamoCore/Utilities/Configurations.cs

```diff
@@ -230,7 +230,7 @@ public class Configurations
 
         #region Backup
 
-        public static string BackupFileNamePrefix = "recent_new_file_";
+        public static string BackupFileNamePrefix = "recent_new_file";
```

Comment:
@riteshchandawar is suggesting to use "home". 
I will change it to "backup". Anyway it will be easy to change it to other names.